### ISSUE TITLE
Enhance debugging by printing user code class name as default operator name

### ISSUE THREAD

**Issue by twalthr** [stratosphere/stratosphere#490]
It would be great if the UDF's user code class name would be chosen as default name of operators instead of e.g. `<Unnamed CoGrouper>`. Giving each operator a name manually is quite inconvenient when creating a plan.

If the constructor of each Builder would contain

```
this.name = udf.getUserCodeClass().getSimpleName();
if (this.name.length() == 0 ) this.name = DEFAULT_NAME;
```

The logs become more readable. E.g.

```
02/21/2014 14:38:46:    CHAIN CoGroup(CustomersWithNoOrders) -> Map(FilterFlaggedCustomers) (4/4)
```

**Comment by StephanEwen** [comment]
That is the case in the new Java api. We can also add this to the old one.
Am 21.02.2014 14:44 schrieb "twalthr" notifications@github.com:

**Comment by StephanEwen** [comment]
Fixed in 762d50670a8e97d976184f528907814b75b299a5
